plan/config.go

```diff
@@ -104,10 +120,35 @@ func parseBehavior(d string) Behavior {
 		ClientAxis: clientAxis,
 		ParamsAxes: values,
 	}
-
 	return behavior
 }
 
+func parseSkipBehavior(d string) (string, []Filter, error) {
+	pair := strings.SplitN(d, "=", 2)
+	if len(pair) != 2 {
+		return "", nil, fmt.Errorf("Invalid input: %s", d)
+	}
+	key := strings.ToLower(pair[0])
+	rawFilters := strings.Split(pair[1], ",")
+	filters := make([]Filter, 0, len(rawFilters))
+	for _, rawFilter := range rawFilters {
+		rawMatches := strings.Split(rawFilter, "+")
+		filter := Filter{
+			AxisMatches: make(map[string]string),
+		}
+		for _, rawMatch := range rawMatches {
+			tuple := strings.SplitN(rawMatch, ":", 2)
+			if len(tuple) != 2 {
+				return "", nil, fmt.Errorf("Invalid filter definition: %s in input %s", rawMatch, d)
+			}
+			tuple = trimCollection(tuple)
+			filter.AxisMatches[tuple[0]] = tuple[1]
+		}
+		filters = append(filters, filter)
+	}
```

Comment:
As discussed in person, this would never happen because of way we're handling errors.